views/dashboard/analytics.ejs

```diff
@@ -245,6 +245,14 @@
                     </td>
                     <td data-title="Newsletter Status"><%= element.stats.newsletterStatus; %></td>
                     <td data-title="Action">
+                        <% if (element.stats.postContentTrackedLinks.length > 0) { %>
```

Comment:
This will cause an issue as it wasn't present in the previous versions. A safety check is necessary.

Reply:
Done.